features/dd-sdk-android-session-replay-compose/consumer-rules.pro

```diff
@@ -43,10 +43,16 @@
 -keepclassmembers class androidx.compose.ui.graphics.AndroidImageBitmap {
      <fields>;
 }
--keepclassmembers class coil.compose.ContentPainterModifier {
+-keep class coil.compose.ContentPainterModifier {
      <fields>;
 }
--keepclassmembers class coil.compose.AsyncImagePainter {
+-keep class coil.compose.AsyncImagePainter {
      <fields>;
 }
+-keepclassmembers class androidx.compose.foundation.layout.PaddingElement{
```

Comment:
in the changes above we are _sometimes_ replacing usages of `keepclassmembers` to just `keep`, so is using it here won't cause any issues?

Reply:
actually it depends on the usage, if in the code we want to check the class instance like this:
```
 if (ComposeReflection.WrappedCompositionClass?.isInstance(composition) == true) {

}
```
then only keeping the class members is not enough, we need to keep the class itself as well, otherwise we can just stick with `keepclassmembers`